vendor.mod

```diff
@@ -103,20 +103,22 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.31.0
 	go.opentelemetry.io/otel/sdk v1.31.0
 	go.opentelemetry.io/otel/trace v1.31.0
-	golang.org/x/mod v0.22.0
-	golang.org/x/net v0.36.0
-	golang.org/x/sync v0.11.0
-	golang.org/x/sys v0.31.0
-	golang.org/x/text v0.22.0
-	golang.org/x/time v0.6.0
+	golang.org/x/mod v0.24.0
+	golang.org/x/net v0.39.0
+	golang.org/x/sync v0.13.0
+	golang.org/x/sys v0.32.0
+	golang.org/x/text v0.24.0
+	golang.org/x/time v0.11.0
 	google.golang.org/genproto/googleapis/api v0.0.0-20241021214115-324edc3d5d38
```

Comment:
- This one is almost merged and will get these out of your way; https://github.com/moby/moby/pull/49777

Reply:
☝️ It's merged, so if you rebase, most of these will be gone 👍